### ISSUE TITLE
Can the accuracy of the timestamp be improved?

### ISSUE THREAD

**Issue by czkoko** [ggerganov/whisper.cpp#255]
The timestamp of whisper is not very accurate. 
The following is the comparison between Microsoft Cognitive Services Speech and whisper.
```
1                                    
00:00:00,120 --> 00:00:01,379 (Microsoft)    
[00:00:00.000 --> 00:00:02.000] (whisper)
2
00:00:02,120 --> 00:00:06,320 (Microsoft)  
[00:00:02.000 --> 00:00:07.500] (whisper)
```

**Comment by misutoneko** [comment]
Yes, this would be much appreciated. I suppose you are using the large model?
I've found the smaller models to be more inaccurate.

**Comment by ggerganov** [comment]
The timestamp precision is a limitation of the model. You would need some sort of pre/post-processing to improve the timestamps. But at the moment it is not clear what is the best approach.

**Comment by pneyrinck** [comment]
Apparently, this work has been done to improve time stamps. https://github.com/jianfch/stable-ts
